File: 19th/코딩테스트 기초반/week2/17388번_와글와글 숭고한.py
s, k, h = map(int, input().split())

if (s + k + h) >= 100:  # s + k + h가 100 이상이면 'OK' 출력
    print('OK')
else:
    if min(s, k, h) == s:  # s, k, h중 최솟값이 s와 같으면 'Soongsil' 출력
        print('Soongsil')
    elif min(s, k, h) == k:  # s, k, h중 최솟값이 k와 같으면 'Korea' 출력
        print('Korea')
    else:  # s, k, h중 최솟값이 h와 같으면 'Hanyang' 출력
        print('Hanyang')

File: 20th/코딩테스트 기초반/week2/17388번_와글와글 숭고한.cpp
#include <iostream>
#include <algorithm> // min 함수를 사용하기 위함

using namespace std;

int main() {
    int S, K, H;
    
    cin >> S >> K >> H;
    
    if (S + K + H >= 100) cout << "OK"; // 합이 100이 넘으면 OK 출력
    else {
        if (min({S, K, H}) == S) cout << "Soongsil"; // 최솟값이 S이면 Soongsil 출력
        else if (min({S, K, H}) == K) cout << "Korea"; // 최솟값이 K이면 Korea 출력
        else cout << "Hanyang"; // 최솟값이 H이면 Hanyang 출력
    }
}